File: C Programs/reverse.c
#include <stdio.h>
int main() {
  int n,n1,s=0,r;
  printf("Enter a number\n");
  scanf("%d",&n);
  n1=n;
  while(n>0)
  {
      r=n%10;
      s=s*10+r;
      n=n/10;
  }
  printf("Reverse of the  number = %d\n",s);
  return 0;
}

File: Java Programs (BlueJ)/Project/reverse.java
import java.util.*;
class reverse
{
    public static void main(String args[])
    {
        int i,l;
        String s,s1="";
        char ch;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a sentence");
        s=sc.nextLine();
        s=s+" ";
        l=s.length();
        for(i=0;i<l;i++)
        {
            ch=s.charAt(i);
            if(ch!=' ')
            {
               s1=ch+s1;
            }
            else
            {
             System.out.print(s1+" ");
             s1=""; 
            }
        }
    }
}